R/normalize-sp.R

```diff
@@ -33,35 +22,11 @@ polygonData_sp <- function(obj) {
 polygonData.Polygon <- polygonData_sp
 #' @export
 polygonData.Polygons <- polygonData_sp
-#' @export
-polygonData.SpatialPolygons <- polygonData_sp
-
-#' @export
-polygonData.SpatialPolygonsDataFrame <- function(obj) {
-  if (length(obj@polygons) > 0) {
-    polygonData(sp::polygons(obj))
-  } else {
-    warning("Empty SpatialPolygonsDataFrame object passed and will be skipped")
-    structure(list(), bbox = obj@bbox)
-  }
-}
```

Comment:
Please revert

----------------

Until `{sp}` is removed, these don't hurt anything for now.

`maybe_as_sf()` doesn't force the output to be `{sf}` types when returned.

If the conversion was required, we should remove the S3 methods.